packages/webpack/test/e2e-policy-gen.spec.js

```diff
@@ -0,0 +1,63 @@
+const test = require('ava')
```

Comment:
if you're interested in adding types for the execution context, an example is in `publish.spec.js`:  https://github.com/LavaMoat/LavaMoat/blob/naugtur/weboack-policy-generation/scripts/publish.spec.js#L53

Reply:
I've got a pending ugly merge in tests here, not gonna make it harder for now.